WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -1888,6 +1894,39 @@ private extension ProductFormViewController {
     }
 }
 
+// MARK: Action - Show Subscription Free trial Settings
+//
+private extension ProductFormViewController {
+    func editSubscriptionFreeTrialSettings() {
```

Comment:
Nit: maybe prefix this with "show" or "navigateTo" instead of "edit` could clarify the intention.

```suggestion
    func showSubscriptionFreeTrialSettings() {
```

Reply:
Done in a108d7d52f6c8938741055c41ee07204e7bcb8ba